vectordb_bench/backend/clients/redis/config.py

```diff
@@ -0,0 +1,16 @@
+from pydantic import SecretStr
+from ..api import DBConfig
+
+class RedisConfig(DBConfig):
+    name: str = "" 
```

Comment:
Same as Chorma, I don't see any usage of param name, please remove it if it's not essentail for connection with the server.